examples/icm20948.toit

```diff
@@ -0,0 +1,29 @@
+// Copyright (C) 2021 Toitware ApS. All rights reserved.
```

Comment:
`accel_gyro.toit` instead, I'd expect examples to have more than one over time and be named after what they are examples of (the scope of the sensor should be clear by the package name)

Reply:
good point.
done.